api/src/main/resources/emails/welcomeemail/content.html

```diff
@@ -86,31 +86,33 @@
       background-color: #302C71;
       font-size: 16px;
       font-weight: 500;
-      line-height: 19px;
       color: #FFFFFF;
-      display: flex;
+      display: inline-block;
       justify-content: space-around;
       align-items: center;
+      text-align: center;
+      line-height: 52px;
       text-transform: uppercase;
       text-decoration: none;
+      cursor: pointer;
     }
   </style>
 </head>
 <body>
-  <img src="assets/logo-registration-non-signed-in.svg">
+  <img src="${HEADER_IMG}">
   <div class="welcome-message">Welcome to the All of Us Researcher Workbench!</div>
-  <div><div class="text user-field">Your Username: </div><div class="user-info">{{USERNAME-INPUT}}</div></div>
-  <div><div class="text user-field">Your Temporary Password: </div><div class="user-info">{{PASSWORD-INPUT}}</div></div>
+  <div><div class="text user-field">Your Username: </div><div class="user-info">${USERNAME}</div></div>
+  <div><div class="text user-field">Your Temporary Password: </div><div class="user-info">${PASSWORD}</div></div>
   <br>
   <div class="text">Quick guide on how to get started:</div>
   <a class="card" href="https://myaccount.google.com/signinoptions/two-step-verification">
-    <div class="bullet">1</div>
+    <img class="bullet" src="${BULLET_1}">
```

Comment:
I suggested to @blrubenstein that we just use CSS for this, as the bullets seem pretty simple. I think we should minimize images as I'd people to have them disabled in their email by default.

Reply:
Please address or reply on the comment if you do not plan to do this.